packages/synapse-react-client/src/components/DataGrid/StartGridSession.tsx

```diff
@@ -13,175 +19,213 @@ export interface StartGridSessionProps {
   onSessionChange?: (session: GridSession) => void
   onReplicaChange?: (replicaId: number | null) => void
   onPresignedUrlChange?: (url: string) => void
-  query?: string
+  show?: boolean
+}
+
+export type StartGridSessionHandle = {
+  handleStartSession: (request: CreateGridRequest) => void
+  handleLoadSession: (sessionId: string) => void
 }
```

Comment:
Update ImperativeHandle signature to support 2 cases:

- Start from a `CreateGridRequest`
- Load from a sessionId